src/deepqmc/wf/paulinet/paulinet.py

```diff
@@ -438,8 +407,6 @@ def forward(self, rs):  # noqa: C901
                 else psi * torch.exp(cusp_same + cusp_anti)
             )
         if self.jastrow:
-            J = self.jastrow(*edges)
+            J = self.jastrow(dists_nuc, dists_elec)
             psi = psi + J if self.return_log else psi * torch.exp(J)
-        if self.omni:
-            self.omni.forward_close()
```

Comment:
Are we certain this isn't needed? I thought it shouldn't be needed, but then randomly, I’d get crashes, apparently the ID was actually the same for a different tensor. Now that I think of it, perhaps because we use the sum of the two id for the total id? Maybe a tuple of them would actually work.

Reply:
Good catch with the sum, this could clearly cause ambiguent IDs. If this works properly I think the forward close shouldn't be needed.